
From: Christoph Hellwig <hch@lst.de>

Seem like devfsd expects it in /dev/cpu/microcode and Debian users do
as well.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/i386/kernel/microcode.c |    1 +
 1 files changed, 1 insertion(+)

diff -puN arch/i386/kernel/microcode.c~fix-devfs-name-for-microcode-driver arch/i386/kernel/microcode.c
--- 25/arch/i386/kernel/microcode.c~fix-devfs-name-for-microcode-driver	2004-08-30 00:05:13.033245440 -0700
+++ 25-akpm/arch/i386/kernel/microcode.c	2004-08-30 00:05:13.038244680 -0700
@@ -485,6 +485,7 @@ static struct file_operations microcode_
 static struct miscdevice microcode_dev = {
 	.minor		= MICROCODE_MINOR,
 	.name		= "microcode",
+	.devfs_name	= "cpu/microcode",
 	.fops		= &microcode_fops,
 };
 
_
